.github/workflows/linkcheck.yml

```diff
@@ -17,5 +17,13 @@ jobs:
           key: cache-lychee-${{ github.sha }}
           restore-keys: cache-lychee-
 
+      # accept 429 only for PR runs where secret isn't available
       - name: Check links
-        run: nix run nixpkgs#lychee -- --verbose docs README.md
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
According to AI read-only by default, so doesn't need further specificaiton

Reply:
This thing was supposed to cache the results and they were supposed to get stored by GH, so we don't strain outside resources. Need to investigate if that's working.

Reply:
Looks like it's working, just sometimes due to the cache works, it is not available to other branches, etc.?